ci/build/test-wheels.sh

```diff
@@ -48,11 +48,12 @@ function retry {
 
 if [[ "$platform" == "linux" ]]; then
   # Install miniconda.
-  PY_WHEEL_VERSIONS=("36" "37" "38" "39")
+  PY_WHEEL_VERSIONS=("36" "37" "38" "39" "311")
```

Comment:
I believe we never do test-wheels for linux - so the codepath never gets triggered for this. 
We could remove this change to keep it minimal.